pycpfcnpj/mask.py

```diff
@@ -0,0 +1,14 @@
+
+from . import cpf
+from . import cnpj
```

Comment:
Avoid relative imports. Use absolute imports.

Reply:
Ps.: I know that the project has a lot of them, but at that time I wasn't aware that's a bad practice. ;) 
That's the beauty of refactoring. :)